resources/views/app.blade.php

```diff
@@ -16,7 +16,7 @@
         @vite(['resources/js/app.tsx', "resources/js/Pages/{$page['component']}.tsx"])
         @inertiaHead
     </head>
-    <body class="font-sans antialiased">
+    <body class="font-sans antialiased bg-wight">
```

Comment:
?

Reply:
it's do a white background like in reference, it was in reference